data/defaults.yaml

```diff
@@ -236,6 +236,13 @@ prometheus::statsd_exporter::package_name: 'statsd_exporter'
 prometheus::statsd_exporter::service_name: 'statsd_exporter'
 prometheus::statsd_exporter::user: 'statsd-exporter'
 prometheus::statsd_exporter::version: '0.17.0'
+prometheus::systemd_exporter::download_extension: 'tar.gz'
```

Comment:
please add the defaults in the class itself, not in hiera. That's an old pattern we don't want to use anymore. adding it in the class enables puppet-strings to pick up the data and put it into the REFERENCE.md

Reply:
Noted, the code have been changed accordingly. :+1: